openpower/patches/nicole-patches/petitboot/0002-Disable-discover-boot-devices-on-Tatlin-RootFS.patch

```diff
@@ -0,0 +1,31 @@
+From 3dd18e438b3e75ea317522d0213f72c8893e0740 Mon Sep 17 00:00:00 2001
+From: Artem Senichev <a.senichev@yadro.com>
+Date: Fri, 9 Oct 2020 10:07:10 +0300
+Subject: [PATCH] Disable discover boot devices on Tatlin RootFS
+
+Disable the search for grub.cfg files on root0/root1 partitions of
+the Tatlin boot disk.
+
+Signed-off-by: Artem Senichev <a.senichev@yadro.com>
+---
+ discover/device-handler.c | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/discover/device-handler.c b/discover/device-handler.c
+index d85f1af..acfe7f2 100644
+--- a/discover/device-handler.c
++++ b/discover/device-handler.c
+@@ -189,7 +189,9 @@ struct discover_boot_option *discover_boot_option_create(
+ 
+ static int device_match_uuid(struct discover_device *dev, const char *uuid)
+ {
+-	return dev->uuid && !strcmp(dev->uuid, uuid);
++	return dev->uuid && !strcmp(dev->uuid, uuid) &&
++		strcmp(dev->uuid, "59616472-6f21-726f-6f74-302150415254") &&
```

Comment:
Could you please define these constants with meaningfult names like `tatlin_root0`/`tatlin_root1` ?